swri_gpu_monitor/package.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package format="2">
+  <name>swri_gpu_monitor</name>
+  <version>0.0.0</version>
```

Comment:
This version number should be the same as every other package in the repository.